### PR TITLE
NPM Package version 0.0.2

### PR DESCRIPTION
**Pull request by arunkumars08**

### CODE REVIEW

**Comment by arunkumars08** [comment]
@humaton Please review the changes.

File: src/app/stack/stack-analyses.service.ts [hunk 1]
```diff
@@ -16,7 +16,8 @@ export class StackAnalysesService {
     private http: Http,
     // private logger: Logger,
```
Comment (humaton) [inline src/app/stack/stack-analyses.service.ts]: Remove commented code please

File: src/app/stack/stack-analyses.service.ts [hunk 2]
```diff
@@ -16,7 +16,8 @@ export class StackAnalysesService {
...
   ) {
-    this.stackAnalysesUrl =  ' http://bayesian-api-bayesian-staging.b6ff.rh-idev.openshiftapps.com/api/v1/'; //apiUrl;
+    // console.log('This is the link', FABRIC8_STACK_API_URL);
```
Comment (humaton) [inline src/app/stack/stack-analyses.service.ts]: Remove commented code please

**Comment by arunkumars08** [comment]
@humaton Changes made. Please review.

**Pull request merged**